Docs 3398: Readme formatting (#2269)
MultiSafepay/docs: content/integrations/lightspeed.md

order: 101
hidden: false
parentDoc: 62a9a54abde254065ee92a5c
excerpt: "Technical manual for installing and configuring MultiSafepay's free app for Lightspeed."
slug: 'lightspeed'
---

<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Lightspeed.svg" width="50" align="right" style="margin: 20px; max-height: 75px"/>

<a class="suggestEdits" style="display: inline-flex; border-radius: 5px; padding: 10px 20px; margin: 10px; font-size: 1rem; background-color: #DFEBF6; color: #0a59a1; text-decoration: none;" href="https://lightspeed.multisafepay.com/changelog" target="_self"><span>Changelog</span></a>
<br>

</details>
> ⚠️ Action required

> > We recommend [upgrading to the latest version](/lightspeed/#upgrading-to-the-new-app) as soon as possible.

Reviewer: ```suggestion
> We recommend [upgrading to the latest version](//docs/lightspeed#how-to-upgrade-to-the-new-app) as soon as possible.
```

# Prerequisites

> - [MultiSafepay account](/getting-started/guide/)

Reviewer: ```suggestion
- [MultiSafepay account](/docs/getting-started-guide/)
```

> - A MultiSafepay [site API key](/tools/multisafepay-control/get-your-api-key)

Reviewer: ```suggestion
- A MultiSafepay [site API key](/docs/sites#site-id-api-key-and-security-code)
```

- The app only supports one account per webshop

# How to install 

> **Tip!** Make sure you have a backup of your production environment, and that you test the plugin in a staging environment.

1. Sign in to your Lightspeed <<glossary:backend>>.
2. Go to **Apps** on the left-hand side of the dashboard.
3. Search for the **MultiSafepay payments app**.
4. Click on the app, and then on **Install app** in the top-right corner.  
5. In the dialog, approve the permissions required for the app.  
   You are redirected to Lightspeed – [MultiSafepay: How to log in](https://lightspeed.multisafepay.com/install).
6. In the **Setup** page:

>     - Enter your email address, [account ID and site API key](/sites/#site-id-api-key-and-secure-code).

Reviewer: ```suggestion
    - Enter your email address, [account ID and site API key](/docs/sites#site-id-api-key-and-secure-code).
```

You don't need to manually update the app. 

## Upgrades

We recommend upgrading from our deprecated core integration to the Lightspeed app.

<details id="how-to-upgrade-to-the-new-app">
<summary>How to upgrade to the new app</summary>
<br>

> 1. In the [Lightspeed app manual](/lightspeed/#installation), follow the steps to install the app.

Reviewer: ```suggestion
1. In the [Lightspeed app manual](/docs/lightspeed#how-to-install, follow the steps to install the app.
```
